src/mugs.js

```diff
@@ -295,7 +295,47 @@ define([
             rawControlXML: {
                 presence: 'optional',
                 lstring: 'Raw XML'
-            }
+            },
+            dataParent: {
+                lstring: 'Data Parent',
+                visibility: function(mug) {
+                    function recFunc(mug) {
+                        if (!mug) {
+                            return true;
+                        } else if (!mug.options.possibleDataParent) {
+                            return false;
+                        }
+                        return recFunc(mug.parentMug);
+                    }
+                    if (mug.__className === "Item") {
```

Comment:
An "Item" can never be a parent mug, and it's parent will never be a possible data parent, so this check seems unnecessary. Correct?